ament_index_cpp/QUALITY_DECLARATION.md

```diff
@@ -121,7 +121,13 @@ A description of how coverage statistics are summarized from this page, can be f
 
 ### Performance [4.iv]
 
-`ament_index_cpp` does not conduct performance tests.
+An environment variable defines the prefix paths of such resource indices and the API has a time complexity of `O(n)` where `n` is the number of prefix paths.
```

Comment:
@dirk-thomas meta: one could argue that we should have tests that backup that claim.

Reply:
How would you create a test to actually do that? What kind of thresholds would you use?

Reply:
I'd probably benchmark queries with increasing values of `n` and check if the result of a linear regression _reasonably_ (within one or two standard deviations) explains the measurements. We would need a big index to see through OS noise though, I give you that. 

And actually, now that I think about it, that's the only kind of performance test that would make sense here.

Reply:
Measuring enough different N trying to figure out if the complexity is linear would be possible. (Even though this sounds extremely costly to me.)